io.github.swanux.hbud.yml

```diff
@@ -0,0 +1,139 @@
+app-id: io.github.swanux.hbud
+runtime: org.gnome.Platform
+runtime-version: '41'
+sdk: org.gnome.Sdk
+command: io.github.swanux.hbud
+
+finish-args:
+  - --share=network
+  - --socket=wayland
+  - --socket=x11
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=host
```

Comment:
why host for a music player?

Reply:
`xdg-music` is probably more appropriate

Reply:
Restricted access to `xdg-music` and `xdg-videos`.

Reply:
On second thought though, I think it's necessary to provide access to at least the whole `home` dir, in order to be able to use this as a default music and video player.

So settling with `filesystem=home` if it's okay.